### DISCUSSION TITLE
Installation on Windows 11 through using terminal

### DISCUSSION THREAD

**Discussion by sedanurgamze** [stella-cv/stella_vslam#560] [Q&A]
Hello, I want to install stella_vslam on my Windows Computer. I do not have enough knowledge about coding, still learning it from basic commands. So, is there any way to share installation which is more specific or is there anyone who can help me to install it on my local computer? Thank you for your help from now on! Have a great day!

**Comment by Skwangles** [comment]
The installation instructions are here
https://stella-cv.readthedocs.io/en/latest/installation.html

For Windows user, please install the dependencies and stella_vslam with [SocketViewer support](https://stella-cv.readthedocs.io/en/latest/installation.html#subsection-dependencies-socketviewer) on [Windows Subsystem for Linux (WSL)](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux).
Alternatively, you can try to use Docker, which is a relatively simple method otherwise
